### PR TITLE
fix: [DEL-3563]: filter active account for rebroadcast iterator

### PR DESCRIPTION
**Pull request by jennyjames21**
You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/31577)

### CODE REVIEW

**Comment by jennyjames21** [comment]
trigger messagecheck

**Comment by jennyjames21** [comment]
trigger messagecheck

File: 400-rest/src/main/java/io/harness/iterator/DelegateTaskRebroadcastIterator.java [hunk 1]
```diff
@@ -95,7 +96,10 @@ public void registerIterators(int threadPoolSize) {
             .targetInterval(Duration.ofSeconds(DELEGATE_TASK_REBROADCAST_TIMEOUT))
             .acceptableNoAlertDelay(Duration.ofSeconds(15))
             .acceptableExecutionTime(Duration.ofSeconds(30))
-            .entityProcessController(new AccountLevelEntityProcessController(accountService))
+            .filterExpander(query
```
Comment (vikas-harness) [inline 400-rest/src/main/java/io/harness/iterator/DelegateTaskRebroadcastIterator.java:99]: Pls test it once as in past I have seen issues with OR condition in filter expander.

**Review by vikas-harness — APPROVED** [review]

File: 400-rest/src/main/java/io/harness/iterator/DelegateTaskRebroadcastIterator.java (hunk 1 above)
Comment (jennyjames21) [reply 400-rest/src/main/java/io/harness/iterator/DelegateTaskRebroadcastIterator.java:99]: vishal, not going with hot fix for this. Merge to develop, currently in QA. 
Also tested in QA, Active account, InActive account, account with no licenseInfo, account with licenseInfo

**Pull request closed without merging**